sql/ddl_internal.sql

```diff
@@ -209,8 +212,14 @@ BEGIN
 
     column_type = _timescaledb_internal.dimension_type(main_table, column_name, num_slices IS NULL);
 
+    IF interval_length IS NOT NULL THEN
+        interval_length_actual = _timescaledb_internal.time_interval_specification_to_internal(
+            column_type, interval_length, INTERVAL '1 month', 'interval_length',
```

Comment:
When having default values like this, we should preferably have it defined in one location, so that a change does not risk making the default value inconsistent across different parts of the code. Now the `1 month` interval is defined in at least two places.

We could wrap this in a function call that hard codes the default value.

Reply:
Done.